### PR TITLE
Design LDAP wizard to better integrate into the design

### PR DESCRIPTION
**Pull request by MorrisJobke**
- mostly replaces gray background with white one

As discussed today with @blizzz and @jancborchardt 

That is how it looks now :)

bildschirmfoto 2016-03-30 um 14 51 34
bildschirmfoto 2016-03-30 um 14 51 45
bildschirmfoto 2016-03-30 um 14 53 15
bildschirmfoto 2016-03-30 um 14 53 40

### CODE REVIEW

File: apps/user_ldap/css/settings.css [hunk 1]
```diff
@@ -208,3 +208,21 @@ select[multiple=multiple] + button {
...
+}
+
+#ldapSettings ul.ui-tabs-nav .ui-tabs-active,
+#ldapSettings ul.ui-tabs-nav .ui-tabs-active > a {
+	background-color: white;
+}
+
+#ldapSettings div.ui-accordion-content {
+	background: white;
+}
```
Comment (LukasReschke) [inline apps/user_ldap/css/settings.css]: Newline? :stuck_out_tongue_winking_eye:

**Comment by jancborchardt** [comment]
@MorrisJobke I assumed you also remove the box around it (as well as some of the additional boxes inside) and the blue bar which should not be used?

**Comment by MorrisJobke** [comment]
> @MorrisJobke I assumed you also remove the box around it (as well as some of the additional boxes inside) and the blue bar which should not be used?

Would be a lot more changes I don't want to add now.

**Comment by jancborchardt** [comment]
Ok. Well, it looks better for sure. Would be great if those changes could also be done in the future. ;)

**Comment by blizzz** [comment]
fancy. what's the status? review or developing?

**Comment by MorrisJobke** [comment]
> review or developing?

Depends on @jancborchardt

File: apps/user_ldap/css/settings.css (hunk 1 above)
Comment (MorrisJobke) [inline apps/user_ldap/css/settings.css]: BTW: It's solved now :P

**Comment by jancborchardt** [comment]
Well, looks better, so fine by me.

**Comment by LukasReschke** [comment]
:+1:

**Comment by LukasReschke** [comment]
@jancborchardt @blizzz

**Comment by blizzz** [comment]
:+1:

**Pull request merged**